colcon_ros/task/catkin/build.py

```diff
@@ -85,7 +85,8 @@ def add_arguments(self, *, parser):  # noqa: D102
             'share' / self.context.pkg.name / 'catkin_env_hook'
         for file_extension in file_extensions:
             additional_hooks += sorted(
-                custom_hooks_path.glob('*.' + file_extension))
+                custom_hooks_path.glob(
```

Comment:
This seems strictly more complicated than string concatenation, having to mentally parse three concepts rather than one.  For the sake of learning, what are you protecting against that I'm missing?

Reply:
As of Python 3.6 this will boil down to: `f'*.{file_extension}`

As soon as Python 3.5 support is dropped the existing pattern of a trailing `.format_map(locals())` will be replaced with a leading `f`.

Reply:
Thanks. It's great to hear that we're going to be giving 3.6 first class support in Colcon (and hopefully ROS2?)